Lombiq.DataTables/Lombiq.DataTables.csproj

```diff
@@ -42,8 +42,8 @@
     <PackageReference Include="ClosedXML" Version="0.96.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Include="Nito.AsyncEx.Tasks" Version="5.1.2" />
-    <PackageReference Include="OrchardCore.Contents" Version="1.4.0" />
-    <PackageReference Include="OrchardCore.Module.Targets" Version="1.4.0" />
+    <PackageReference Include="OrchardCore.Contents" Version="1.5.0" />
+    <PackageReference Include="OrchardCore.Module.Targets" Version="1.5.0" />
```

Comment:
The below `OrchardCore.Lucene` reference should be changed to `OrchardCore.Search.Lucene`. Look up all the other Lucene references in OSOCE too.